app/src/main/java/org/piramalswasthya/sakhi/ui/home_activity/disease_control/leprosy/suspected/from/SuspectedLeprosyFromViewModel.kt

```
@@ -0,0 +1,168 @@
+                            it.placeOfDeath = leprosyScreenCache.placeOfDeath
+                            it.placeOfDeathId = context.resources.getStringArray(R.array.death_place).indexOf(leprosyScreenCache.placeOfDeath)
+                            it.otherPlaceOfDeath = leprosyScreenCache.otherPlaceOfDeath
+                            if (it.processed != "N") it.processed = "U"
+                            it.syncState = SyncState.UNSYNCED
+                            benRepo.updateRecord(it)
+                        }
+                    }
+
+                    leprosyRepo.updateLerosyScreening(leprosyScreenCache)
+                    _state.postValue(State.SAVE_SUCCESS)
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)